pycbc/io/live.py

```diff
@@ -263,10 +265,16 @@ def __init__(self, ifos, coinc_results, **kwargs):
         for ifo in self.psds:
             psd = self.psds[ifo]
             kmin = int(kwargs['low_frequency_cutoff'] / psd.delta_f)
+            if kwargs['high_frequency_cutoff']:
```

Comment:
@MarlinSchaefer @titodalcanton I'm not sure we need to modify the psd we give to bayestar do we? Assumin git covers at least the range intended?

Reply:
I do not think this is required either.

Reply:
@lpsinger does BAYESTAR care if the PSD extends past the `f_final` frequency?